lagom1-java-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/lagom/javadsl/ServiceLocatorModule.scala

```diff
@@ -3,6 +3,7 @@ package com.typesafe.conductr.bundlelib.lagom.javadsl
 import javax.inject.Singleton
 
 import com.lightbend.lagom.javadsl.api.ServiceLocator
+import com.typesafe.conductr.bundlelib.scala.Env
```

Comment:
It is, on line 18.

That used to resolve implicitly to `com.typesafe.conductr.bundlelib.lagom.javadsl.Env`, which inherited from `com.typesafe.conductr.bundlelib.scala.Env`. With the former removed, an explicit import of the latter had to be added.